### DISCUSSION TITLE
API Scan library for new files

### DISCUSSION THREAD

**Discussion by DeltaTango69** [immich-app/immich#10708] [General]
Hello,

it seems to be that the api for scan library request a full scan. My wish is only to scan for new files. Maybe it is possible to revise the payload for that

```
payload = json.dumps({
  "refreshAllFiles": True,
  "refreshModifiedFiles": True
maybe
```

`"refreshNewFiles"`

Is this possible?

**Comment by bo0tzz** [comment]
The payload from the docs is just an example, you can set those fields to False as well.

**Reply by DeltaTango69** [reply]
Ok, thanks - but what is the payload for scanning only for new files in the external library?

**Reply by DeltaTango69** [reply]
![image](https://github.com/immich-app/immich/assets/82575543/6ec0ce4d-2d73-408e-9983-fdf8180f56ae)
